lib/event_icon_builder.rb

```diff
@@ -0,0 +1,36 @@
+class EventIconBuilder
```

Comment:
A lot of the stuff in this class will be moved out to an abstract class that this and other "Builders" will inherit from.  But since we only have one, for right now, I figured not to over engineer it.  Once we get a second route/object going, I'll make this more generalizable.